.github/workflows/qa.yaml

```diff
@@ -108,7 +118,7 @@ jobs:
           cargo install grcov
       - name: Run tests (with coverage collection)
         run: |
-          set -eu
+          set -eux
```

Comment:
Oh, yeah a leftover... Although I still feel it should be the default in our CI, since github is not really telling us what's doing and this allows better grained logs.

However, let me get this out for now